uisdk/src/main/java/com/karhoo/uisdk/screen/booking/booking/BookingPaymentPresenter.kt

```diff
@@ -22,9 +22,23 @@ class BookingPaymentPresenter(view: BookingPaymentMVP.View,
 
     init {
         attachView(view)
+        getPaymentProvider()
```

Comment:
Would be good to add a unit test for this logic. 
e.g. You can create a method and put all these three lines in that method. Then in unit test, you can call that method and ensure this method `paymentsService.getPaymentProvider()` is called.

Reply:
I think this should be possible by adding an additional verify in the two new tests that have already been added.